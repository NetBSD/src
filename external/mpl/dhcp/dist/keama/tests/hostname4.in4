# host name config

# subnet4 declaration
subnet 10.5.5.0 netmask 255.255.255.224 {
    range 10.5.5.5 10.5.5.10;
}

# host declaration
host test1 {
    hardware ethernet 00:0B:FD:32:E6:FA;
    fixed-address 10.5.5.1;
}

# host declaration using a longer name
host test2.example.com {
    hardware ethernet 00:07:0E:36:48:19;
    fixed-address 10.5.5.2;
}
