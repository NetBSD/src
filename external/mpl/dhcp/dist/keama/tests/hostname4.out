{
  # host name config
  # subnet4 declaration
  /// This configuration declares some subnets but has no interfaces-config
  /// Reference Kea #245
  "Dhcp4": {
    "subnet4": [
      {
        "id": 1,
        "subnet": "10.5.5.0/27",
        "pools": [
          {
            "pool": "10.5.5.5 - 10.5.5.10"
          }
        ]
      }
    ],
    "host-reservation-identifiers": [
      "hw-address"
    ],
    "reservation-mode": "global",
    "reservations": [
      # host declaration
      {
        "hostname": "test1",
        "hw-address": "00:0b:fd:32:e6:fa",
        "ip-address": "10.5.5.1"
      },
      # host declaration using a longer name
      {
        "hostname": "test2.example.com",
        "hw-address": "00:07:0e:36:48:19",
        "ip-address": "10.5.5.2"
      }
    ]
  }
}
